recipes/R/Retail - GradientBoosting/R/applicationEvaluator.R

```diff
@@ -55,7 +57,7 @@ applicationEvaluator <- setRefClass("applicationEvaluator",
       if(!is.null(tenantId)){
         if(any(names(df) == '_id')) {
           #Drop id, eventType, timestamp and rename columns
-          df <- df[,-c(1,2,3)]
+          df <- df[,-c(12,13,14)]
```

Comment:
_id, eventType and timestamp are no longer at places 1,2,3.
Now they are on 12,13 and 14.